libyul/optimiser/DataFlowAnalyzer.cpp

```diff
@@ -237,6 +239,14 @@ optional<YulString> DataFlowAnalyzer::memoryValue(YulString _key) const
 		return nullopt;
 }
 
+optional<YulString> DataFlowAnalyzer::keccakValue(YulString _start, YulString _length) const
+{
+	if (YulString const* value = util::valueOrNullptr(m_state.environment.keccak, make_pair(_start, _length)))
```

Comment:
```suggestion
	if (YulString const* value = valueOrNullptr(m_state.environment.keccak, make_pair(_start, _length)))
```
There's also `valueOrDefault` in `CommonData.h` (if this is even where `util::valueOrNullptr` is even implemented that could potentially simplify this function by just having
```
return valueOrDefault(m_state.environment.keccak, make_pair(_start, _length), nullopt);
```
Note that I haven't actually looked into this at great detail, so I may be wrong :)

Reply:
It at least doesn't work right away, because the vale is a YulString and in the positive case we would have to convert it to `option<YulString>` first.

Reply:
Ah yes, true. In that case, just get rid of the `util::` qualifier for consistency.